### PR TITLE
samples: wifi: add wifi tag for samples

### PR DESCRIPTION
**Pull request by nordic-piks**
Extend list of paths used to check if wifi
samples should be executed.

### CODE REVIEW

File: scripts/ci/tags.yaml [hunk 1]
```diff
@@ -97,7 +97,7 @@ posix:
   files:
     - zephyr/lib/posix/
 
-# cbprintf:
-#    files:
-#        - zephyr/lib/os/cbprintf*
-#        - zephyr/lib/posix
+wifi:
+  files:
+    - nrf/drivers/wifi
```
Comment (krish2718) [inline scripts/ci/tags.yaml:102]: `modules/hostap/` and `../modules/lib/hostap` too.

File: scripts/ci/tags.yaml [hunk 2]
```diff
@@ -97,7 +97,9 @@ posix:
...
 
-# cbprintf:
-#    files:
-#        - zephyr/lib/os/cbprintf*
-#        - zephyr/lib/posix
+wifi:
+  files:
+    - nrf/drivers/wifi/
+    - nrf/modules/hostap/
+    - nrf/modules/lib/hostap/
```
Comment (krish2718) [inline scripts/ci/tags.yaml:104]: This is outside `nrf`, hence the `..`

File: scripts/ci/tags.yaml [hunk 3]
```diff
@@ -97,7 +97,9 @@ posix:
...
+    - zephyr/drivers/wifi/
```
Comment (krish2718) [inline scripts/ci/tags.yaml:105]: This isn't needed for NCS

File: scripts/ci/tags.yaml (hunk 1 above)
Comment (nordic-piks) [reply scripts/ci/tags.yaml]: Thanks, fixed.
The root folder is NCS, so it would be `modules/lib/hostap`

File: scripts/ci/tags.yaml (hunk 3 above)
Comment (nordic-piks) [reply scripts/ci/tags.yaml]: Thanks, removed.

File: scripts/ci/tags.yaml (hunk 1 above)
Comment (nordic-piks) [reply scripts/ci/tags.yaml]: And `modules/hostap` is actually at `sdk-nrf`

**Comment by nordic-piks** [comment]
Updated after evaluation of:
https://github.com/nrfconnect/sdk-nrf/blob/main/.github/test-spec.yml#L497-L501
https://github.com/nrfconnect/sdk-zephyr/blob/main/.github/test-spec.yml#L246-L248
https://github.com/nrfconnect/sdk-nrfxlib/blob/main/.github/test-spec.yml#L168-L173

Please note that the content is similar and syntax is different - the root path is NCS, all dependencies - from west modules - are stated at one place, tests and samples should not be included here, they are handled separately (but they will be build if changed)

**Review by VivekUppunda — APPROVED** [review]

**Review by bama-nordic — APPROVED** [review]

**Review by bjda — APPROVED** [review]

**Review by D-Triveni — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]

**Review by muraliThokala — APPROVED** [review]

**Comment by nordic-piks** [comment]
Just rebased after NCS 2.7

**Pull request merged**